tesseract/tesseract_planning/tesseract_process_managers/include/tesseract_process_managers/process_input.h

```diff
@@ -68,13 +68,13 @@ struct ProcessInput
   const tesseract::Tesseract::ConstPtr tesseract;
 
   /** @brief Instructions to be carried out by process */
-  const Instruction& instruction;
```

Comment:
Perhaps you could comment in the PR as to why this change was necessary for posterity

Reply:
Updated